typescript/sdk/src/token/checker.ts

```diff
@@ -36,8 +40,31 @@ export class HypERC20Checker extends ProxiedRouterChecker<
   HypERC20App,
   HypTokenRouterConfig
 > {
+  constructor(
+    multiProvider: MultiProvider,
+    app: HypERC20App,
+    configMap: ChainMap<HypTokenRouterConfig>,
+    ismFactory?: HyperlaneIsmFactory,
+    logger?: pino.Logger,
+    readonly warpDeployConfig?: WarpRouteDeployConfig | null,
```

Comment:
whats the difference between this config and configMap?

Reply:
Why do we have `HypTokenRouterConfig` and `WarpRouteDeployConfig` as separate types?

Reply:
Removed the new param/field/etc. as the existing configMap should be enough